app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -76,45 +104,74 @@ class AamvaStateMaintenanceWindow
-        { cron: '0 5 * * Sun', duration_minutes: 7 * 60 },
       ],
-      # NM: "Sunday mornings." (not modeling; too vague)
+      'ND' => [
+        # Wednesday around 7:30 pm to 7:35 pm ET
+        { cron: '30 19 * * Wed', duration_minutes: 5 },
+        # 3rd Sunday of month, 5 minutes anytime between midnight and noon.
+      ],
+      'NM' => [
+        # Sundays 8:00 am. to noon ET.
+        { cron: '0 8 * * Sun', duration_minutes: 4 * 60 },
+      ],
+      'NV' => [
+        # Tuesdays to Sundays: 2:00 am. to 3:15 am. ET
+        # MW FIXME: This wraps around, does Tue-Sun get parsed correctly?
+        { cron: '0 2 * * Tue-Sun', duration_minutes: 1.25 * 60 },
+      ],
       'NY' => [
         # Sunday maintenance 8 pm. to 9 pm. ET.
         { cron: '0 20 * * Sun', duration_minutes: 60 },
       ],
+      'OH' => [
+        # Daily 4:00 am. to 4:30 am. ET
+        { cron: '0 4 * * *', duration_minutes: 30 },
+      ],
+      'OR' => [
+        # Sunday 7:30 am. to 9:00 am. ET.
+        { cron: '30 7 * * Sun', duration_minutes: 1.5 * 60 },
+      ],
       'PA' => [
-        # Sunday maintenance may occur, often between 5:30 am. & 7:00 am. ET
-        { cron: '30 5 * * Sun', duration_minutes: 90 },
+        # Sunday 5:00 am. to 7:00 am. ET.
+        { cron: '0 5 * * Sun', duration_minutes: 2 * 60 },
+      ],
+      'RI' => [
+        # Either 3rd or 4th Sunday of each month, 7:30 am. to 10:00 am. ET.
+        { cron: '30 7 * * Sun#3', duration_minutes: 2.5 * 60 },
+        { cron: '30 7 * * Sun#4', duration_minutes: 2.5 * 60 },
       ],
       'SC' => [
-        # Sunday maintenance from 7:00 pm. to 10:00 pm. ET.
-        { cron: '0 19 * * Sun', duration_minutes: 3 * 60 },
+        # Sunday 6:00 pm. to 10:00 pm. ET.
+        { cron: '0 18 * * Sun', duration_minutes: 4 * 60 },
+      ],
+      'TN' => [
+        # Last Sunday of every month from 11:00 pm Sunday to 2:00 am. Monday ET
+        { cron: '0 23 * * Sun#4', duration: 3 * 60 },
+        { cron: '0 23 * * Sun#5', duration: 3 * 60 },
       ],
       'TX' => [
-        # Downtime on weekends between 9 pm ET to 7 am ET.
-        { cron: '0 21 * * Sat,Sun', duration_minutes: 10 * 60 },
+        # Saturday 9:00 pm. to Sunday 7:00 am. ET.
+        { cron: '0 21 * * Sat', duration_minutes: 10 * 60 },
+      ],
+      'UT' => [
+        # 3rd Sunday of every month 1:00 am. to 9:00 am. ET
+        { cron: '0 1 0 0 Sun#3', duration_minutes: 8 * 60 },
       ],
       'VA' => [
-        # Sunday morning maintenance 3:00 am. to 5 am. ET.
-        { cron: '0 3 * * Sun', duration_minutes: 120 },
-        # Daily maintenance from 5 am. to 5:30 am.
+        # Daily 5:00 am. to 5:30 am. ET
         { cron: '0 5 * * *', duration_minutes: 30 },
-        # "Might not respond for short spells, daily between 7 pm  and 8:30 pm." (not modeling this)
+        # Sunday morning maintenance 3:00 am. to 5 am. ET.
+        { cron: '0 3 * * Sun', duration_minutes: 2 * 60 },
```

Comment:
This is the same as it was before, but I moved it to match document order. (And changed to the `2 * 60` format for readability.)